math/vecops/test/vecops_rvec.cxx

```diff
@@ -788,7 +788,7 @@ TEST(VecOps, UnqiueCombinationsSingleVector)
 
 TEST(VecOps, PrintCollOfNonPrintable)
 {
-   auto code = "class A{};ROOT::VecOps::RVec<A> v(1);cling::printValue(&v);";
+   auto code = "class A{};ROOT::VecOps::RVec<A> v(1);v;";
```

Comment:
Remove the trailing `';'` or no printing will happen!

Reply:
ouch, how did this test pass then? :/

Reply:
Nobody is looking at the output. It would even pass if it "prints" `RVec<A>& @0x12345`

Reply:
ah yes of course, thanks :)

Reply:
I am lost: this https://github.com/root-project/root/blob/master/math/vecops/test/vecops_rvec.cxx#L791 prints correctly and the test checks that this does not result in something which does not compile... The new version does not print anything on the other hand.

Reply:
the test does not really check the print output